images/windows/Windows2022-Readme.md

```diff
@@ -523,7 +523,6 @@ All other versions are saved but not installed.
 | Package Name               | Version                                                                                                                                                                                           |
 | -------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Android Command Line Tools | 8.0                                                                                                                                                                                               |
-| Android Emulator           | 31.2.9                                                                                                                                                                                            |
```

Comment:
this does not seem correct, it needs investigation

Reply:
Hi @mikhailkoliada  ,  i have missed it. Closing this issue and will start new deployement.